components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -8,11 +8,13 @@
 
 """
 
+import traceback
 from datetime import datetime
 from omero.cli import BaseControl, CLI
 import platform
 import sys
 import os
+import pwd
```

Comment:
This import is breaking the Web CLI plugin on Windows - see https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-deploy-win2012/100/console

```
6360
Error loading: C:\jenkins\workspace\OMERO-5.1-merge-deploy-win2012\OMERO.server-5.1.3-m1-193-12fd548-ice35-b557\lib\python\omero\plugins\web.py
Traceback (most recent call last):
  File "C:\jenkins\workspace\OMERO-5.1-merge-deploy-win2012\OMERO.server-5.1.3-m1-193-12fd548-ice35-b557\lib\python\omero\cli.py", line 1361, in loadpath
    execfile(str(pathobj), loc)
  File "C:\jenkins\workspace\OMERO-5.1-merge-deploy-win2012\OMERO.server-5.1.3-m1-193-12fd548-ice35-b557\lib\python\omero\plugins\web.py", line 17, in <module>
    import pwd
ImportError: No module named pwd
```